api/routes/course.js

```diff
@@ -182,4 +184,18 @@ router.get('/', courseController.getCoursesByAttribute);
  */
 router.get('/carousel/', courseController.getCoursesForCarousel);
 
+router.post(
+  '/subscribe/:courseId',
+  jwtGuard,
+  refreshToken,
+  courseController.postSignatureUserCourse
+);
+
+router.get(
```

Comment:
Pls write swagger documentation on this route.

Reply:
Done